content/android/tutorials/_index.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: "Tutorials"
+title: "WorldWind Android Tutorials"
 date: 2017-06-30T14:51:58-04:00
 draft: false
 uselisttemplate: true
 ---
 
 ## Tutorials
 
```

Comment:
There's some odd spacing in the Android tutorials opening paragraph. It doesn't show up in the final product, but ought to be addressed anyway.